draft-selander-lake-authz.md

```diff
@@ -535,30 +535,45 @@ V receives the voucher response from W over the secure connection. If the receiv
 # REST Interface at W
 
 The interaction between V and W is enabled through a RESTful interface exposed by W.
+This RESTful interface MAY be implemented using either HTTP or CoAP.
 V SHOULD access the resources exposed by W through the protocol indicated by the scheme in LOC_W URI.
-In case the scheme indicates "https", V SHOULD perform a TLS handshake with W and use HTTP.
+
+## Scheme "https" {#scheme-https}
+In case the scheme indicates "https", V MUST perform a TLS handshake with W and use HTTP.
+If the public key of V, PK_V, is a signature key that can be used in a TLS handshake, V SHOULD use the certificate containing PK_V to authenticate to W as a client.
+In case the public key of V, PK_V, is a static DH key that cannot be used in a TLS handshake, V MUST first perform a TLS handshake with W using the available compatible keys.
+V MUST then perform an EDHOC handshake on top of the TLS session, using and proving the possesion of PK_V.
+Performing the EDHOC handshake is only necessary if V did not use PK_V in the TLS handshake with W.
+
```

Comment:
"PK_V" terminology is removed from the rest of the draft and we need to harmonize this. The terminology PK_V is not used in EDHOC. There may be more than one public key of V. Elsewhere we use "public key of CRED_V" which perhaps is better since it refers to a specific authentication credential. 
Proposal:
OLD
>If the public key of V, PK_V, is a signature key that can be used in a TLS handshake, V SHOULD use the certificate containing PK_V to authenticate to W as a client.
In case the public key of V, PK_V, is a static DH key that cannot be used in a TLS handshake, V MUST first perform a TLS handshake with W using the available compatible keys.
V MUST then perform an EDHOC handshake on top of the TLS session, using and proving the possesion of PK_V.
Performing the EDHOC handshake is only necessary if V did not use PK_V in the TLS handshake with W.

NEW

If the authentication credential CRED_V can be used in a TLS handshake, e.g. an X.509 certificate of a signature public key, then V SHOULD use it to authenticate to W as a client.
If the authentication credential CRED_V cannot be used in a TLS handshake, e.g. if the public key is a static Diffie-Hellman key, then V SHOULD first perform a TLS handshake with W using available compatible keys. V MUST then perform an EDHOC handshake over the TLS connection proving to W the possession of the private key corresponding to CRED_V. Performing the EDHOC handshake is only necessary if V did not authenticate with CRED_V in the TLS handshake with W.

Reply:
fixed: using now the proposed text